ui/src/components/Swap/DialogSwap/AssetDialog.jsx

```diff
@@ -96,12 +35,12 @@ const AssetDialog = ({ type, setSelectedAsset }) => {
 
   return (
     <div className="flex flex-col gap-4 p-5 overflow-auto ">
-      {parsedAssets?.map(item => (
+      {filteredAssets?.map(item => (
```

Comment:
it's defined as an array
```suggestion
      {filteredAssets.map(item => (
```

Reply:
Done